linode/databasemysql/framework_datasource_schema.go

```diff
@@ -24,6 +25,9 @@ var frameworkDatasourceSchema = schema.Schema{
 		"label": schema.StringAttribute{
 			Description: "A unique, user-defined string referring to the Managed Database.",
 			Computed:    true,
+			Validators: []validator.String{
+				stringvalidator.LengthBetween(3, 32),
```

Comment:
I think we don't have to apply the validator to computed-only attribute because the user is not allowed to config these attributes.

Reply:
Wait does this schema not correspond to [https://www.linode.com/docs/api/databases/#managed-mysql-database-create](https://www.linode.com/docs/api/databases/#managed-mysql-database-create) ?

Reply:
It's a data source schema, so it correspond to the GET endpoints. In the Terraform Framework, data source schema can't be shared with resources (at least for now). 

And additionally, user can't configured a computed-only attribute (i.e. `Computed: true` and doesn't contain `Optional` or `Required` field), even in a resource schema, so the validator is not needed on these attributes.

Reply:
Ahhh I see, that kicks out quite a few of these no? Actually I think that leaves only two where the label is actually manually configured?

Reply:
Note: we have discussed and resolved this in a private chat. Resolving this issue.